packages/assets-controllers/src/NftController.test.ts

```diff
@@ -1048,15 +1188,23 @@ describe('NftController', () => {
         "Unable to verify ownership. Possibly because the standard is not supported or the user's currently selected network does not match the chain of the asset in question.",
       );
     });
+
+    // it('handle unset selectedAccount', async function () {
```

Comment:
Do we still need this test? It's a new one, and it's being commented, so I'd say no?